Assets/Tests/InputSystem/CoreTests_MouseEvents.cs

```diff
@@ -0,0 +1,199 @@
+#if UNITY_6000_4_OR_NEWER
```

Comment:
Great you added automated tests for all the events! This way we protect ourselves from regressions in this area. Similarly I think there is a lot to gain from having them covered in the sample as well.